### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by wojciechowskidawid**

### CODE REVIEW

File: examples/hello-world/hello_world.robot [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    Execute Command          machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl
+
+    Execute Command          sysbus LoadELF @${CURDIR}/binaries/hello_world/hello_world.ino.elf
+
+*** Test Cases ***
+Should Print Brightness Sequence
+    Create Machine
+    Create Terminal Tester    ${UART}
+
+    Execute Command           machine EnableProfiler "${CURDIR}/hello_world.dump"
```
Comment (PiotrZierhoffer) [inline examples/hello-world/hello_world.robot:23]: we should actually use the `@` notation, just like in LoadELF, whenever we provide a path. You can change that in other robot files as well, maybe just a PR to verify the tests and merge without a review

File: examples/hello-world/hello_world.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+shutdown_renode()
+connect_renode() # this sets up a log file, and clears the simulation (just in case)
+tell_renode('using sysbus')
+tell_renode('mach create')
+tell_renode('machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl')
+tell_renode('sysbus LoadELF @binaries/hello_world/hello_world.ino.elf')
+
+tell_renode('uart0 CreateFileBackend @uart.dump true')
+tell_renode('logLevel 3')
+tell_renode('machine EnableProfiler "metrics.dump"')
```
Comment (PiotrZierhoffer) [inline examples/hello-world/hello_world.py:38]: `@` here as well (see the note in .robot file)

File: examples/hello-world/hello_world.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+import os
+from renode_colab_tools import metrics
+os.environ['PATH'] = os.getcwd()+"/renode:"+os.environ['PATH']
+
+# %%
+!mkdir -p binaries/hello_world && cd binaries/hello_world && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/hello_world/hello_world.ino.elf # fetch prebuilt binaries
+
+# %% [markdown]
+"""## Run a hello-world example in Renode"""
```
Comment (PiotrZierhoffer) [inline examples/hello-world/hello_world.py:24]: the hello-world

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]
Minor suggestions, otherwise lgtm

File: examples/hello-world/hello_world.py (hunk 2 above)
Comment (PiotrZierhoffer) [reply examples/hello-world/hello_world.py:38]: This is not resolved

**Review by PiotrZierhoffer — APPROVED** [review]
Fix this one thing and merge

**Review by mateusz-holenko — APPROVED** [review]

**Pull request merged**